Gemfile.lock

```diff
@@ -34,10 +53,10 @@ GEM
     bigdecimal (1.4.4)
     chroma (0.2.0)
     claide (1.0.3)
-    cocoapods (1.10.1)
+    cocoapods (1.10.2)
```

Comment:
It wasn't necessary for the successful behavior of this PR to bump CocoaPods. I did it to research https://github.com/wordpress-mobile/release-toolkit/pull/289 and left it in here since it's a patch update.